### PR TITLE
Oscal app inputs

### PR DESCRIPTION
**Pull request by omarabed15**
Saves input files in the database.
Relates the app inputs to the app version through a new model (AppInput).
Iterates through the inputs in app.yaml. Currently only processes OSCAL JSON input files, but is extensible for future additional support.
Adds Components and Statements to the Component Library upon app loading.
Adds Components and Statements to the project when a new project is created. Only adds Components if they have a Statement. Otherwise, the Component gets added to the Component Library but won't get added to the Project.

Input format in the app.yaml file is expected as:

```
input:
- id: oscal_components
  name: OSCAL Components
  type: oscal
  path: data/oscal_components.json
  group: OSCAL
```

This PR supercedes #1335 .

### CODE REVIEW

**Comment by gregelin** [comment]
"Adds Components and Statements to the project when a new project is created." Suuhweet, @omarabed15

File: guidedmodules/migrations/0055_auto_20210121_2312.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.0.11 on 2021-01-21 23:12
```
Comment (davidpofo) [inline guidedmodules/migrations/0055_auto_20210121_2312.py:1]: Need to only have one migration file that is affecting app version instead of 3.

Comment (omarabed15) [reply guidedmodules/migrations/0055_auto_20210121_2312.py]: Fixed! Good catch

**Review by davidpofo — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 2]
```diff
@@ -23,6 +23,14 @@ Add lightweight-ato to default apps so users can get started easier.
...
 * Add Lightweight-ato apps to default apps.
 
+v0.9.1.49.2 (January 22, 2021)
+------------------------------
+
+Adds support for OSCAL component and statement input for Compliance Apps.
+(Currently only supports OSCAL JSON inputs.)
+Adds statements to project upon project creation.
+Keeps track of app inputs by relating them to the app version.
+
```
[the remark this answers is not in the document]
Comment (omarabed15) [reply CHANGELOG.md:33]: @davidpofo I've updated this here, and will make the same updates to the govready-q-docs repo.

**Review by davidpofo — APPROVED** [review]
lgtm PR party 1/22

**Pull request merged**